ros2interface/ros2interface/command/interface.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.command import add_subparsers
+from ros2cli.command import CommandExtension
+from ros2cli.verb import get_verb_extensions
+
+
+class InterfaceCommand(CommandExtension):
```

Comment:
This class doesn't have a docblock and as a consquence their is no description (which is the first line of the docblock) when calling `ros2 --help`.

Reply:
Added in https://github.com/ros2/ros2cli/pull/434